### DISCUSSION TITLE
Error in submission page

### DISCUSSION THREAD

**Discussion by Cosijopiii** [esa/SpOC2#6] [Quantum Constellations Q&A] [answered]
Hello, I am unable to submit my JSON file due to the following error:

I don't know if there is a bug; I use the challenge and problem ID that the page problem indicates to me.

![image](https://user-images.githubusercontent.com/7028517/229433612-35286c6e-9048-4ad7-b635-c3b62b1465e6.png)

**Comment by EmBlazquez** [comment, marked as answer]
Hi @Cosijopiii,

Please use the following identifiers:

```python
"problem": "quantum-communications-constellations",
"challenge": "spoc-2-quantum-communications-constellations"
```

**Reply by Cosijopiii** [reply]
Thanks, it works!
